build.sh

```diff
@@ -270,10 +270,10 @@ function runWattsi {
 }
 
 runWattsi $HTML_TEMP/source-whatwg-complete $HTML_TEMP/wattsi-output
-$QUIET || cat $HTML_TEMP/wattsi-output.txt | grep -v '^$' # trim blank lines
-
-if [ "$WATTSI_RESULT" != "0" ]; then
-  $QUIET && exit $WATTSI_RESULT
+if [ "$WATTSI_RESULT" == "0" ]; then
+    $QUIET || cat $HTML_TEMP/wattsi-output.txt | grep -v '^$' # trim blank lines
+else
+  cat $HTML_TEMP/wattsi-output.txt | grep -v '^$' # trim blank lines
```

Comment:
I think the necessity of repeating either this line or the $WATTSI_RESULT check if moved out to just below runWattsi is why I left it silent in #58. This seems tolerable, though.